### PR TITLE
Added Lyriek

### PR DESCRIPTION
**Pull request by BartWillems**

### CODE REVIEW

**Comment by palfrey** [comment]
I've got a few concerns about this one:
- No CI (not vital, but I always get a bit concerned when projects don't have it)
- No build instructions, and getting `cargo run` to work required quite a lot of things installed
- Doesn't actually seem to work. I'm seeing "no active player" found when playing stuff with QuodLibet. It's plausible I'm missing a step, or it only supports some players, but none of that is documented AFAIK.

@BartWillems Thoughts?

**Comment by BartWillems** [comment]
@palfrey 

I've imported this repo from my Gitlab account, so there is CI, but it's currently only for Gitlab. 
I'm planning to rework this project quite q bit, so Github CI is one of the first things on the road map.

The readme is indeed quite lacking, I'll add build instructions tomorrow.

Lyriek connects to [MPRIS](https://wiki.archlinux.org/index.php/MPRIS), so if your player doesn't interface with it, then it won't work.

**Comment by BartWillems** [comment]
@palfrey I've added the github actions CI, updated the readme & added the badge to this readme.

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 
 * [enginesound](https://github.com/DasEtwas/enginesound) — A GUI and command line application used to procedurally generate semi-realistic engine sounds. Featuring in-depth configuration, variable sample rate and a frequency analysis window.
 * [indiscipline/zrtstr](https://github.com/indiscipline/zrtstr) — A command line utility for checking if stereo wav files are faux-stereo (i.e. have identical channels) and converting such files to mono. [<img src="https://api.travis-ci.org/indiscipline/zrtstr.svg?branch=master">](https://travis-ci.org/indiscipline/zrtstr)
+* [Lyriek](https://github.com/bartwillems/lyriek) — A multi-threaded GTK 3 application for fetching the lyrics of the current playing song. <img src=https://github.com/bartwillems/lyriek/workflows/Rust/badge.svg>
```
Comment (palfrey) [inline README.md:144]: Please correct the build badge syntax to use markdown syntax.

Comment (BartWillems) [reply README.md]: Fixed

Comment (palfrey) [reply README.md]: Now that it's using the Markdown syntax, the build has picked up the missing branch parameter. Please see https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter and add the branch parameter for the main/master branch of the repository

File: README.md [hunk 2]
```diff
@@ -141,6 +141,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 
 * [enginesound](https://github.com/DasEtwas/enginesound) — A GUI and command line application used to procedurally generate semi-realistic engine sounds. Featuring in-depth configuration, variable sample rate and a frequency analysis window.
 * [indiscipline/zrtstr](https://github.com/indiscipline/zrtstr) — A command line utility for checking if stereo wav files are faux-stereo (i.e. have identical channels) and converting such files to mono. [<img src="https://api.travis-ci.org/indiscipline/zrtstr.svg?branch=master">](https://travis-ci.org/indiscipline/zrtstr)
+* [Lyriek](https://github.com/bartwillems/lyriek) — A multi-threaded GTK 3 application for fetching the lyrics of the current playing song. [<img src=https://github.com/bartwillems/lyriek/workflows/Rust/badge.svg?branch=master>](https://github.com/BartWillems/lyriek/actions)
```
Comment (palfrey) [inline README.md:144]: This is missing quotes around the `src` parameter

Comment (BartWillems) [reply README.md]: fixed

**Pull request merged**